apps/synapse-oauth-signin/src/LoginComponent.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect } from 'react'
+import { useSearchParams } from 'react-router-dom'
+import { SynapseClient, useSynapseContext } from 'synapse-react-client'
+
+export function LoginComponent() {
```

Comment:
Existing component, just pulled out into its own file. I don't think it's used, but I'm trying not to make many functional changes in this PR.